demo/simple/index.html

```diff
@@ -0,0 +1,22 @@
+<p>Simple demos that show FPS performance for some common actions. In
```

Comment:
It'd be better to include the preamble:

```html
<!DOCTYPE html>
<html lang="en">
<meta charset="utf-8">
<meta name="viewport" content="width=device-width, initial-scale=1">
<title>Simple virtual-scroller demos</title>
```

This is mostly best-practice stuff that won't matter in this particular case:

- Avoids quirks mode
- Declare to screenreaders, scrapers, etc. that you're using English
- Avoids any content being interpreted as the legacy default charset (windows-1252)
- Makes it appear reasonable on mobile
- Makes the tab have a reasonable title

Reply:
Done.